README.md

```diff
@@ -109,27 +115,30 @@ services:
 Now we run `runserver` manually to have more control over restarts:
 
 ```sh
-docker compose exec django bash
+docker-compose up -d django
+docker-compose exec django bash
 root$ python manage.py runserver 0.0.0.0:8000
 ```
 
 ### Initial Setup
 
+Migrate DB:
```

Comment:
Switched the order of migrate and createsuperuser because you'll get fewer warnings in red.